gh-pages/docs/device_selection.md

```
@@ -0,0 +1,86 @@
+or device.
+
+## List available devices
+
+```bash
+dune exec -- sarek-device-info
+```
+
+This prints every detected device across all backends with its framework and
+capabilities, e.g.:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)